src/main/java/Main.java

```diff
@@ -0,0 +1,24 @@
+import sums.impl.ExecutorServiceSum;
+import sums.impl.ForkJoinSum;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+public class Main {
+    public static void main(String[] args) throws InterruptedException, ExecutionException {
+        List<Integer> list = new ArrayList<>();
+        for (int i =0; i<1_000_000; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < 1_000_000; i++) {
```
don't forget about codestyle